one_day_intern/settings.py

```diff
@@ -40,6 +43,8 @@
     'rest_framework',
 
     'main',
+    'users',
+    'assessor',
```

Comment:
Hi, Indi! If possible, please move the assessor functionality to be under users, such that all functionalities that supports login and registration are placed under one django app.

Reply:
Hi Jo, to clarify the assessor functionality doesn't include the assessor's login and registration. It includes the dashboard and other functionalities the assessor will be able to access. For the assessor authentication it's already in the users application. Hope I cleared things up! :)

Reply:
Noted, my mistake.